addons/web/static/src/core/popover/popover.js

```diff
@@ -111,36 +111,10 @@ export class Popover extends Component {
 
         useForwardRefToParent("ref");
         this.popoverRef = useRef("ref");
-
-        let shouldAnimate = this.props.animation;
         this.position = usePosition("ref", () => this.props.target, {
             onPositioned: (el, solution) => {
-                (this.props.onPositioned || this.onPositioned.bind(this))(el, solution);
-                if (this.props.arrow && this.props.onPositioned) {
-                    this.onPositioned.bind(this)(el, solution);
-                }
-
-                // opening animation
-                if (shouldAnimate) {
-                    shouldAnimate = false; // animate only once
-                    const transform = {
-                        top: ["translateY(-5%)", "translateY(0)"],
-                        right: ["translateX(5%)", "translateX(0)"],
-                        bottom: ["translateY(5%)", "translateY(0)"],
-                        left: ["translateX(-5%)", "translateX(0)"],
-                    }[solution.direction];
-                    this.position.lock();
-                    const animation = el.animate(
-                        { opacity: [0, 1], transform },
-                        this.constructor.animationTime
-                    );
-                    animation.finished.then(this.position.unlock);
-                }
-
-                if (this.props.fixedPosition) {
-                    // Prevent further positioning updates if fixed position is wanted
-                    this.position.lock();
-                }
+                this.onPositioned(el, solution);
+                this.props.onPositioned?.(el, solution);
```

Comment:
Cool for both functions being executed. I saw some rare issues with dropdown not syncing properly with the `useHover` hook including some other manual `Dropdown` in Discuss (see Discuss app sidebar when compact), items are hoverable similarly to chat bubble preview).

So with this change this is likely fixed :)